plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSplitManager.java

```diff
@@ -38,7 +38,7 @@
     private final IcebergTransactionManager transactionManager;
 
     @Inject
-    public IcebergSplitManager(IcebergTransactionManager transactionManager, HiveTableOperationsProvider tableOperationsProvider)
+    public IcebergSplitManager(IcebergTransactionManager transactionManager)
```

Comment:
this is related, because HiveTableOperationsProvider is an implementation detail of `TrinoHiveCatalog` after the refactoring and should not be a part of the constructor for split manager.